fusesoc/vlnv.py

```diff
@@ -161,3 +161,53 @@ def __lt__(self, other):
             other.name,
             other.version,
         )
+
+    def vln_str(self):
+        """Returns a string with <vendor>:<library>:<name>"""
+        return f"{self.vendor}:{self.library}:{self.name}"
+
+
+def compare_relation(vlvn_a, relation, vlvn_b):
```

Comment:
We have total_ordering from functools imported which allows us to specify __eq__ and __lt__ which then allows us to use regular comparisons between vlnv objects. From what I understand, this function does a bit more advanced dependency comparing than just blindly comparing two tuples, but perhaps we want to move part of this code into __lt__.

Reply:
Ok, I see now that it won't work the way I thought